crates/re_sdk/Cargo.toml

```diff
@@ -30,6 +30,23 @@ glam = ["re_components/glam"]
 ## Integration with the [`image`](https://crates.io/crates/image/) crate, plus JPEG support..
 image = ["re_components/image"]
 
+## Support serving a web viewer over HTTP.
+##
+## Enabling this inflates the binary size quite a bit, since it embeds the viewer wasm.
+# When building from source (in the repository), this feature adds quite a bit
+# to the compile time since it requires compiling and bundling the viewer as wasm.
+#
+# You also need to install some additional tools, which you can do by running
+# [`scripts/setup_web.sh`](https://github.com/rerun-io/rerun/blob/main/scripts/setup_web.sh).
```

Comment:
Unclear if ending the doc string for the internal part was intentional or not.
In any case "building from source" is confusing for a newcomer ("aren't all crates build from source???") since they don't know that we cheat and don't build the webviewer from source when they download from crates.io
I think we should briefly mention that here and make the whole thing doc string.


Suggestion:
```
##
## For faster & easier builds, the web-player build comes bundled with the crate.
## Enabling this feature will embed this pre-built web-player.
##
## However, when building from source in the repository, this feature adds quite a bit
## to the compile time since it requires compiling and bundling the viewer as wasm.
## You also need to install some additional tools, which you can do by running
## [`scripts/setup_web.sh`](https://github.com/rerun-io/rerun/blob/main/scripts/setup_web.sh).
```